src/main/java/org/springframework/data/redis/connection/convert/Converters.java

```diff
@@ -569,8 +570,9 @@ enum ClusterNodesConverter implements Converter<String, RedisClusterNode> {
 
 		@Override
 		public RedisClusterNode convert(String source) {
-
-			String[] args = source.split(" ");
+			String filteredSource = source.replace(", ", " ");
```

Comment:
That won't do the job especially when auxilliary fields are appended, then the `port` part is immediately followed by aux fields.